product_code/lagom/authentication_service/impl/src/main/resources/application.conf

```diff
@@ -2,6 +2,8 @@
 #
 play.application.loader = de.upb.cs.uc4.authentication.impl.AuthenticationLoader
 
```

Comment:
Ist glaube ich Sinnvoll in einem Kommentar zu erwähnen, was 
`uc4.authentication.logout = 10`
bedeutet